trustvector.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package ar4si
+
+// TODO(tho) description
+type TrustVector struct {
```

Comment:
Please also add specific reference to the specification where we have inherited the magic numbers!

Reply:
the pointer is already in the README